src/dmd/frontend.h

```diff
@@ -504,6 +504,8 @@ enum class TOK : uint16_t
     __restrict = 257u,
     __declspec = 258u,
     __attribute__ = 259u,
+    __asm__ = 260u,
+    __asm = 261u,
```

Comment:
are you sure these aren't gøˆng tø cause keyword collisions for LDC and GDC in the future with dtoh?

Reply:
There's already `__attribute__`, so the file is already uncompilable (well, it's uncompilable for _other_ reasons as well).

At this point in time, I'm not sure how dtoh will help the compiler, as until it's stable, there's no possibility of bootstrapping it.

Reply:
fair enough.

Reply:
https://issues.dlang.org/show_bug.cgi?id=21941